packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -699,7 +699,7 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       case '#':
       case '*':
         var builder = UpdateVerbBuilder()
-          ..atKey = serverCommitEntry['atKey']
+          ..atKeyObj = AtKey.fromString(serverCommitEntry['atKey'])
```

Comment:
Just looking at this again ... would there ever be situations in which the server commit entry might contain an `atKey` which is invalid?